corehq/apps/case_importer/tests/test_importer_config_compat.py

```diff
@@ -1,4 +1,5 @@
 from __future__ import absolute_import
+from __future__ import unicode_literals
```

Comment:
Failure real. Lines 18 and 38 should be bytestrings. (Unable to comment on those lines.)